sites/footers/deprecated/footers-v8-fr.html

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
These pages are not needed. There are already pages for the working examples of the current footer. Please remove footers-v8-fr and -en.